app/controllers/mixins/generic_show_mixin.rb

```diff
@@ -72,7 +72,7 @@ def show_main
                        :url  => "/#{controller_name}/show_list?page=#{@current_page}&refresh=y"},
                       true)
 
-      drop_breadcrumb(:name => _("%{name} (Summary)") % {:name => @record.name},
+      drop_breadcrumb(:name => _("%{name} (Summary)") % {:name => @record.respond_to?(:name) ? @record.name : @record.description},
```

Comment:
Maybe we have a helper for `name/description`. If not, we could create one. But there should be one I think.